docs/ref/migration-operations.txt

```diff
@@ -150,6 +150,16 @@ a default value to put into existing rows. It does not affect the behavior
 of setting defaults in the database directly - Django never sets database
 defaults and always applies them in the Django ORM code.
 
+.. warning::
+
+    When adding a field with a default value defined, PostgreSQL and likely other
+    databases, will initialize all existing rows with the column's default value.
```

Comment:
That's not true for PostgreSQL 11+

https://www.postgresql.org/about/news/1894/

> Removing the need for ALTER TABLE .. ADD COLUMN .. DEFAULT .. with a not NULL default to rewrite the whole table on execution, which provides a significant performance boost when running this command.

Reply:
That's awesome! Do you think we should keep the note but add a reference to PostgreSQL 11+, @charettes? It should still be an issue with other DBs and older versions of PG. Thanks for the heads up!

Reply:
@caioariede I think the note is still valuable since it's still a problem on other databases as well (e.g. [MySQL < 8](https://mysqlserverteam.com/mysql-8-0-innodb-now-supports-instant-add-column/)). A database agnostic mention about _older databases_ could be enough it might be a source of doc rot in a few years from now.

It feel like there's no way around getting into the version number details for all supported backend. If we go this way we should figure out if it's still and issue on SQLite, MariaDB, and Oracle.

Reply:
Agreed, thanks. I'll update the PR